src/App.jsx

```diff
@@ -14,33 +15,79 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const copyOfGoods = [...goodsFromServer];
+  const [sortType, setState] = useState('');
+  const [isReversed, setReserve] = useState(false);
```

Comment:
The naming convention is typically to name the state variable and its updater function in a way that they correspond to each other for readability and maintainability.
```suggestion
  const [isReversed, setIsReserved] = useState(false);
```